### DISCUSSION TITLE
Add support for the Media Library

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Revisions#921] [Feature Requests]
Let's consider adding support for media files in PublishPress Revisions

https://twitter.com/mor10/status/1199755445616828416

> Number 1 takeaway from interviewing people about using #WordPress to build intranet sites: File management and file versioning is a major issue.

**Comment by stevejburge** [comment]
This plugin has some ideas that overlap very closely with what we do.
https://wordpress.org/plugins/wp-document-revisions/

However, the documents are very separate from the Media Library.

They currently recommend Edit Flow to work with their plugin:
https://ben.balter.com/wp-document-revisions/frequently-asked-questions/

**Comment by JoshJourney** [comment]
Would be a handy addition for file collaborations. In my case would enjoy people being able to work on each other's photos. Many years ago a lad Joey (from the UK) did a [lovely workshop](https://treknature.com/workshops/159270/) on a photo of mine. Original is up top, Joey's is in the middle. We've had many conversations since which led to becoming an administrator of a site he was running for enthusiastic photographers. In my early years of photography "photo revisions" (workshops) gave better insight on how to capture, crop, edit photos. Could see this encouraging others.

**Comment by stevejburge** [comment]
> Does your tool allow us to schedule PDF/media library content as well as page content? I am using Revisionize currently, and it does not offer that capability and our team needs it.
